allennlp/tests/training/trainer_test.py

```diff
@@ -86,12 +88,33 @@ def test_trainer_can_run_cuda(self):
                           cuda_device=0)
         trainer.train()
 
+  # pylint: disable=arguments-differ
```

Comment:
Is this for the call to `forward` below?  I'd say this belongs inside that scope, if that's what this is for.

Reply:
Oops! Fixed. This line escaped during some experimentation with the linter.